src/Exercise07AI.ipynb

```diff
@@ -67,7 +67,7 @@
    "source": [
     "## Progressive Deepening\n",
     "\n",
-    "Da das in `Exercise05AI` implementierte Alpha-Beta-Pruning am effizientesten ist, wenn zuerst die besten Züge untersucht werden, wird nun die Liste der möglichen Züge vorab nach der bisher bekannten Evaluierung jedes Zuges sortiert. Die bisherigen Evaluierungen werden dabei aus dem, in `Exercise06AI` implementierten, Memoisierungs-Cache abgerufen.\n",
+    "Das in `Exercise05AI` implementierte Alpha-Beta-Pruning ist deutlich effizienter (aber nicht zwingend optimal), wenn zuerst die besten Züge untersucht werden[9]. Aus diesem Grund wird nun die Liste der möglichen Züge vorab nach der bisher bekannten Evaluierung jedes Zuges sortiert. Die bisherigen Evaluierungen werden dabei aus dem, in `Exercise06AI` implementierten, Memoisierungs-Cache abgerufen.\n",
```

Comment:
```suggestion
    "Das in `Exercise05AI` implementierte Alpha-Beta-Pruning ist deutlich effizienter (aber nicht zwingend optimal), wenn zuerst die besten Züge untersucht werden [9]. Aus diesem Grund wird nun die Liste der möglichen Züge vorab nach der bisher bekannten Evaluierung jedes Zuges sortiert. Die bisherigen Evaluierungen werden dabei aus dem, in `Exercise06AI` implementierten, Memoisierungs-Cache abgerufen.\n",
```

Reply:
Ist in meinem branch gefixt. Weitere Merge-Conflicts bitte vermeiden